src/metrics_test.go

```diff
@@ -16,33 +16,33 @@ import (
 )
 
 var expectedRawInfoFromSample = map[string]interface{}{
-	"active_defrag_hits":           0,
-	"active_defrag_key_hits":       0,
-	"active_defrag_key_misses":     0,
-	"active_defrag_misses":         0,
-	"active_defrag_running":        0,
-	"aof_current_rewrite_time_sec": -1,
-	"aof_enabled":                  0,
-	"aof_last_bgrewrite_status":    "ok",
-	"aof_last_cow_size":            0,
-	"aof_last_rewrite_time_sec":    -1,
-	"aof_last_write_status":        "ok",
-	"aof_rewrite_in_progress":      0,
-	"aof_rewrite_scheduled":        0,
-	"arch_bits":                    64,
-	"atomicvar_api":                "atomic-builtin",
-	"blocked_clients":              0,
-	"client_biggest_input_buf":     0,
-	"client_longest_output_list":   0,
-	"cluster_enabled":              0,
-	"config_file":                  "",
-	"connected_clients":            1,
-	"connected_slaves":             0,
-	"evicted_keys":                 0,
-	"executable":                   "/Users/newrelic/redis-server",
-	"expired_keys":                 0,
-	"gcc_version":                  "4.2.1",
-	"hz":                           10,
+	"active_defrag_hits":             0,
+	"active_defrag_key_hits":         0,
+	"active_defrag_key_misses":       0,
+	"active_defrag_misses":           0,
+	"active_defrag_running":          0,
+	"aof_current_rewrite_time_sec":   -1,
+	"aof_enabled":                    0,
+	"aof_last_bgrewrite_status":      "ok",
+	"aof_last_cow_size":              0,
+	"aof_last_rewrite_time_sec":      -1,
+	"aof_last_write_status":          "ok",
+	"aof_rewrite_in_progress":        0,
+	"aof_rewrite_scheduled":          0,
+	"arch_bits":                      64,
+	"atomicvar_api":                  "atomic-builtin",
+	"blocked_clients":                0,
+	"client_biggest_input_buf":       0,
+	"client_longest_output_list":     0,
+	"cluster_enabled":                0,
+	"config_file":                    "",
+	"connected_clients":              1,
+	"connected_slaves":               0,
+	"evicted_keys":                   0,
+	"executable":                     "/Users/newrelic/redis-server",
+	"expired_keys":                   0,
+	"gcc_version":                    "4.2.1",
+	"hz":                             10,
```

Comment:
Apologies for the whitespace changes. I believe this is the result of gofmt, so I left them in since they'll probably end up here eventually. I can remove them again if you'd like to keep the PR focused on the real changes